docs/docs/getting-started.server.mdx

```diff
@@ -636,6 +636,33 @@ well.
 
 See [Using MDX with Next.js][next-mdx] for more details.
 
+#### Astro
+
+[Astro](https://astro.build/) is built on [islands architecture](https://docs.astro.build/concepts/islands/),
+allowing you to use React, Vue, Svelte, and more within your MDX files.
```

Comment:
Understood! Just wanted a brief sentence on how Astro supports React, Vue, Svelte and more (rather than a single JSX framework). I can rework to be less sell-y if you prefer 👍